### PR TITLE
Remove graduated custom media headers

### LINKED ISSUE #1488: Remove graduated custom media headers
GitHub Developer v3 API announcement:
https://developer.github.com/changes/2020-04-07-graduated-previews/

This would be a great PR for any new contributor to this repo or a new Go developer.
All contributions are greatly appreciated!

Feel free to volunteer for any issue and the issue can be assigned to you so that others don't attempt to duplicate the work.

Please check out our [CONTRIBUTING.md](https://github.com/google/go-github/blob/master/CONTRIBUTING.md) guide to get started.

Thank you!

### PR DESCRIPTION
**Pull request by wesleimp**
Closes #1488

### CODE REVIEW

File: github/github.go [hunk 1]
```diff
@@ -109,9 +109,6 @@ const (
 	// https://developer.github.com/changes/2018-12-18-interactions-preview/
 	mediaTypeInteractionRestrictionsPreview = "application/vnd.github.sombra-preview+json"
```
Comment (gmlewis) [inline github/github.go:111]: Shouldn't lines 49-51 in github.go be removed too?

Comment (wesleimp) [reply github/github.go:108]: Yes, I missed this one

File: github/github.go [hunk 2]
```diff
@@ -110,6 +107,9 @@ const (
...
 
 	// https://developer.github.com/changes/2019-03-14-enabling-disabling-pages/
+	// https://developer.github.com/changes/2019-02-14-draft-pull-requests/
```
Comment (gmlewis) [inline github/github.go:110]: I don't think you meant to add this one back, did you?

Comment (wesleimp) [reply github/github.go]: No, I didn't. Sorry, my bad

**Comment by nightlark** [comment]
I'll review this later today.

**Comment by gmlewis** [comment]
Thank you, @nightlark!
Merging.

**Pull request merged**